src/main.js

```diff
@@ -103,7 +103,16 @@ var load = loader({
   server: {
     requires: ['cfg', 'router', 'docs'],
     setup: ({cfg, router, docs}) => {
-      let hooksApp = app(cfg.server);
+      let hooksApp = app({
+        port:                   cfg.server.port,
```

Comment:
It looks like publicUrl is missing here?

Reply:
but publicUrl is not being used in tc-lib-app. Right?